Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml

```
@@ -862,6 +862,38 @@
         - ReagentId: Vitamin
           Quantity: 3
 
+- type: entity         
```

trim whitespace here
```suggestion
- type: entity
```

---

Fixed.